tsp_cli_client

```diff
@@ -483,17 +483,22 @@ if __name__ == "__main__":
                 if not configuration_source_set or len(configuration_source_set.configuration_source_set) == 0:
                     print('No configuration sources available')
                 else:
-                    print('  {0}'.format(configuration_source_set.to_string()))
+                    print('Successfully listed configuration sources')
+                    print('-----------------------------------------')
+                    print(response.model.to_json())
                 sys.exit(0)
             else:
                 sys.exit(1)
```

Comment:
Just wondering why we only print message on error in certain cases?

Reply:
The error is printed in the library. For this case see `tsp_client.fetch_configuration_sources()`